### ISSUE TITLE
A novel not showing all chapters and non of them are working!

### ISSUE THREAD

**Issue by bad2066boy** [LNReader/lnreader#783]
### Steps to reproduce

1. I use "ArNovel" Source (It's an arabic source btw)
2. Most of the novels i read on it are fine
3. Except "البداية بعد النهاية" or in English it's named "the beginning after the end"
4. Not all the chapters are there, there are only 11 chapters (out of 400-ish)
5. And non of them are working
6. But in the site all the chapters are there and work as well
7. So i wish you find me a fix for that
8. Thanks

### Expected behavior

I guess to find a fix for that

### Actual behavior

Non of the chapters are working in that novel
And not all of them are there

### LNReader version

1.1.17

### Android version

Android 13

### Device

Xiaomi Poco F3

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with a source, I should be opening an issue in the [sources repository](https://github.com/lnreader/lnreader-sources/issues/new/choose).
- [X] I have updated the app to version **[1.1.17](https://github.com/lnreader/lnreader/releases/latest)**.
- [x] I will fill out all of the requested information in this form.

**Comment by K1ngfish3r** [comment]
Can confirm, `beginning after the end` is 0 chapters. Currently waiting on traitor to resolve this issue

**Comment by K1ngfish3r** [comment]
download this debug build https://github.com/K1ngfish3r/lnreader/actions/runs/6314189542
To download:
1. Sign in to github
2. Click on the link below Artifacts
![image](https://github.com/LNReader/lnreader/assets/26593485/e0a73414-c60f-40c5-96af-06f48561b6ac)

Once installed
More > Settings > Advanced > Clear + Clean Database, Cache, etc and try viewing the novel again

**Comment by rajarsheechatterjee** [comment]
> If this is an issue with a source, I should be opening an issue in the [sources repository](https://github.com/lnreader/lnreader-sources/issues/new/choose).
